c7n/filters/health.py

```diff
@@ -21,43 +22,42 @@ class healthEventFilter(Filter):
 
     schema = type_schema(
             'health-events',
-            required=['eventTypeCodes'],
+            entityValues={'type': 'array', 'items': {'type': 'string'}},
```

Comment:
i think there was a misunderstanding, entityValues isn't a parameter for the filter, its part of the internal logic on the filter. a resource id is an entityValue for server side filtering purposes.